doc/_templates/sidebar-nav-bs.html

```diff
@@ -0,0 +1,11 @@
+<nav class="bd-links" id="bd-docs-nav" aria-label="{{ _('Main navigation') }}">
+  <div class="bd-toc-item active">
+    {{ generate_nav_html("sidebar",
+                         show_nav_level=theme_show_nav_level|int,
+                         maxdepth=theme_navigation_depth|int,
+                         collapse=theme_collapse_navigation|tobool,
+                         includehidden=True,
+                         startdepth=0,
```

Comment:
Ok I see, so adding `startdepth=0` seemed to do the trick! It's the only difference I can spot with the template from the pydata sphinx theme, do you confirm?

https://github.com/pydata/pydata-sphinx-theme/blob/b806673afbcc5e13a68d2a28b1dde92c8729cbd7/src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/sidebar-nav-bs.html#L7

Maybe we should submit that upstream, as an optional configuration?

Reply:
Chris already submitted a PR to this effect but closed it: https://github.com/pydata/pydata-sphinx-theme/pull/536/files

Reply:
It's not exactly the same as your change affects all the pages while he was only addressing that on the landing page. Sounds useful to me!